.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,17 @@
-<!--  Thanks for sending a pull request!  Here are some tips for you -->
+## Pull Request template
```

Comment:
It will be good if this TEMPLATE retains the existing format enhances it as follows:

```
Please, go through these steps before you submit a PR.

**Why is this PR required? What issue does it fix?**:

**What this PR does?**:

**Does this PR require any upgrade changes?**:

**If the changes in this PR are manually verified, list down the scenarios covered:**:

**Any additional information for your reviewer?** : 
_Mention if this PR is part of any design or a continuation of previous PRs_


**Checklist:**
- [ ] Fixes #<issue number>
- [ ] PR Title follows the convention of  `<type>(<scope>): <subject>`
- [ ] Has the change log section been updated? 
- [ ] Commit has unit tests
- [ ] Commit has integration tests
- [ ] (Optional) Are upgrade changes included in this PR? If not, mention the issue/PR to track: 
- [ ] (Optional) If documentation changes are required, which issue on https://github.com/openebs/openebs-docs is used to track them: 


**PLEASE REMOVE BELOW INFORMATION BEFORE SUBMITTING**

The PR title message must follow convention:
   `<type>(<scope>): <subject>`.

Where:
   `type` is defining if release will be triggering after merging submitted changes, details in [CONTRIBUTING.md](../CONTRIBUTING.md).
    Most common types are:
    * `feat`      - for new features, not a new feature for build script
    * `fix`       - for bug fixes or improvements, not a fix for build script
    * `chore`     - changes not related to production code
    * `docs`      - changes related to documentation
    * `style`     - formatting, missing semi colons, linting fix etc; no significant production code changes
    * `test`      - adding missing tests, refactoring tests; no production code change
    * `refactor`  - refactoring production code, eg. renaming a variable or function name, there should not be any significant production code changes


```

Reply:
what could be some good examples of `scope`

single word that best describes where the changes fits like:
- data engine ( localpv, jiva, cstor ...)
- feature (provisioning, backup, restore)
- code component ( webhook, cast, log, or a pkg name)

Probably look at the past month's PRs and come up with some good suggestions on what can be included in the `scope`